tests/unittest/_torch/visual_gen/test_trtllm_serve_e2e.py

```
@@ -247,9 +247,9 @@ def test_health(self, server):
         ],
     )
     def test_t2v_sync(self, server, format_, expected_content_type):
-        """Synchronous text-to-video via POST /v1/videos/generations."""
+        """Synchronous text-to-video via POST /v1/videos/sync."""
         resp = requests.post(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)